### PR TITLE
Adding pull request requirements to the readme.md

### PR DESCRIPTION
**Pull request by Agaba-derrick**
fixes https://github.com/I-TECH-UW/OpenELIS-Global-2/issues/827

### CODE REVIEW

**Review by mozzy11 — CHANGES_REQUESTED** [review]
Thanks @Agaba-derrick . 
For a pull request template 
see [instructions](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository)

ie create a file called ` .github/PULL_REQUEST_TEMPLATE.md`
and add the requirements so that the requirement appear by default in each PR created. 

see example here  https://github.com/google/fhir-data-pipes/blob/master/.github/PULL_REQUEST_TEMPLATE.md

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # OpenELIS Global 2
```
Comment (mozzy11) [inline README.md:1]: Can you get rid of the format changes in the Read me , i dont see any actual change

**Review by mozzy11 — CHANGES_REQUESTED** [review]
@Agaba-derrick , can you get rid of the  format changes in the read me file ??

**Comment by Agaba-derrick** [comment]
> @Agaba-derrick , can you get rid of the format changes in the read me file ??

sure amon it now

**Review by mozzy11 — APPROVED** [review]

**Review by mozzy11 — APPROVED** [review]

**Pull request merged**